content/source/docs/cloud/run/index.html.md

```diff
@@ -47,171 +69,11 @@ There are three ways to run speculative plans:
 
 ## Run States
 
-Each run passes through several stages of action (pending, plan, policy check, apply, and completion), and Terraform Cloud shows the progress through those stages as run states. In some states, the run might require confirmation before continuing or ending; see [Interacting with Runs](#interacting-with-runs) below.
+Each run passes through several stages of action (pending, plan, policy check, apply, and completion), and Terraform Cloud shows the progress through those stages as run states. In some states, the run might require confirmation before continuing or ending; see [Managing Runs: Interacting with Runs](./manage.html#interacting-with-runs) for more information.
```

Comment:
> Each run passes through several stages of action (pending, plan, policy check, apply, and completion), and Terraform Cloud shows the progress through those stages as run states.

Terraform Cloud shows the progress of each run as it passes through each run state (pending, plan, policy check, apply, and completion).

Reply:
Ah, good call. 

Hey, have you seen the "suggestions" interface yet? It's a button with a page that has a plus/minus icon on it, hidden next to the markup helper buttons:

```suggestion
Terraform Cloud shows the progress of each run as it passes through each run state (pending, plan, policy check, apply, and completion). In some states, the run might require confirmation before continuing or ending; see [Managing Runs: Interacting with Runs](./manage.html#interacting-with-runs) for more information.
```

Reply:
Wasn't aware that was there, thanks! That'll come in handy :)